Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Impute"
 uuid = "f7bf1975-0170-51b9-8c5f-a992d46b9575"
 authors = ["Invenia Technical Computing"]
-version = "0.6.8"
+version = "0.6.9"
```

Comment:
You don't need to bump the version here.

Reply:
Done :)